app/views/categories/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="form-style mx-5">
+  <h1>New category</h1>
+  <%= render "form", category: @category %>
+
+  <br>
+
+  
+    <%= link_to "Back to categories", categories_path, class:"btn btn-primary mb-5" %>
+  
+</div>
```

Comment:
- [ ] Please use colors font, colors, etc that were used in the [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
`#3778c2`, `#5fb253`,  `#434b54`